internal/controller/controllers/agentserviceconfig_controller.go

```diff
@@ -605,6 +653,64 @@ func (r *AgentServiceConfigReconciler) newAgentRoute(ctx context.Context, log lo
 	return route, mutateFn, nil
 }
 
+func (r *AgentServiceConfigReconciler) newHTTPRoute(ctx context.Context, log logrus.FieldLogger, instance *aiv1beta1.AgentServiceConfig, serviceToExpose string, description string) (client.Object, controllerutil.MutateFn, error) {
```

Comment:
Is it really necessary to have a separate `description` param here? Any reason not to use `serviceToExpose` in logs? I'd imagine it would be more meaningful, especially given messages like `Failed to get https route for %s service` where I would expect the `%s` to be a service name.